rootfs/opt/fluentd/deis-output/lib/fluent/plugin/out_deis.rb

```diff
@@ -26,6 +26,10 @@ def initialize
       @influx_nsq = nil
       @log_topic = ENV['NSQ_LOG_TOPIC'] || "logs"
       @metric_topic = ENV['NSQ_METRIC_TOPIC'] || "metrics"
+      logs_to_nsq = ENV['SEND_LOGS_TO_NSQ'] || "true"
+      metrics_to_nsq = ENV['SEND_METRICS_TO_NSQ'] || "true"
+      @send_logs_to_nsq = if logs_to_nsq.downcase == 'false' then false else true end
```

Comment:
`logs_to_nsq = ENV['SEND_LOGS_TO_NSQ'].downcase == 'true'`

was also the first thing I thought, problem is, if SEND_LOGS_TO_NSQ is not set, this will throw a exception:

```
2.3.1 :001 > if ENV['SEND_LOGS_TO_NSQ'].downcase == 'false' then false else true end
NoMethodError: undefined method `downcase' for nil:NilClass
	from (irb):1
	from /Users/felipecavalcanti/.rvm/rubies/ruby-2.3.1/bin/irb:11:in `<main>'
```